app/build.gradle

```diff
@@ -51,6 +51,8 @@ dependencies {
     implementation "com.google.android.material:material:1.2.0-alpha02"
     implementation "androidx.constraintlayout:constraintlayout:1.1.3"
     implementation 'com.github.PhilJay:MPAndroidChart:v3.1.0'
+    implementation "com.android.support:preference-v7:28.0.0"
+    implementation "com.android.support:appcompat-v7:28.0.0"

```

Comment:
It seems you've used the deprecated pre-androidx preferences library, can you update to the jetpack one?
https://developer.android.com/jetpack/androidx/releases/preference

Reply:
`implementation "androidx.preference:preference-ktx:$preference_version"`

Reply:
Nice catch, will update.

Reply:
Updated.